src/main/java/org/broadinstitute/consent/http/util/CountryValidator.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.consent.http.util;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.JsonParseException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.util.HashSet;
+import java.util.Set;
+
+public class CountryValidator {
+  private static final String FILEPATH = "assets/ISO-3166-countries.json";
+  private Set<String> countries;
+  public CountryValidator() {
+    try (InputStream is = getClass().getClassLoader().getResourceAsStream(FILEPATH)) {
```

Comment:
I think this should be fine, but an alternative might be to load the file contents into a local cache.